platform/ui/src/components/content/viewportDownloadForm/ViewportDownloadForm.js

```diff
@@ -284,17 +284,27 @@ const ViewportDownloadForm = ({
           className="viewport-preview"
           src={viewportPreview.src}
           alt="Viewport Preview"
+          data-cy="viewport-preview-img"
         />
       </div>
 
       <div className="actions">
         <div className="action-cancel">
-          <button type="button" className="btn btn-danger" onClick={onClose}>
+          <button
+            type="button"
+            data-cy="cancel-btn"
```

Comment:
We already have a `data-cy="cancel-btn"` in `UserPreferencesForm`, I think it's better to rename it to prevent conflicts:

https://github.com/OHIF/Viewers/blob/27e02825a456981158b227e3a26364b9469b8609/platform/ui/src/components/userPreferencesForm/UserPreferencesForm.js#L76-L80

Maybe we can start using some pattern for naming these `data-cy`, what's your opinion @mirnasilva?

Reply:
Because they're in different containers that should never be visible at the same time, it's probably not a huge deal. If the form already has a `data-cy`, we could also key off the button's text after using the container to scope our search.

Reply:
I thought the same thing since `UserPreferencesForm` and `DownloadFile` are both modals, they won't be visible at the same.